devicetypes/technisat/technisat-series-switch.src/technisat-series-switch.groovy

```diff
@@ -0,0 +1,440 @@
+/**
+ *	Copyright 2021 TechniSat
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata  {
+	definition (name: "TechniSat Series switch", namespace: "TechniSat", author: "TechniSat", vid:"generic-switch-power-energy",
+				mnmn: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012',
```

Comment:
`runLocally` should be false. the need to address on commands to endpoint 1 I believe will make this not work locally. you could also just remove the `runLocally`, `minHubCoreVersion`, and `executesCommandsLocally` flags.